packages/dds/tree/.eslintrc.js

```diff
@@ -12,5 +12,21 @@ module.exports = {
 		"@typescript-eslint/no-namespace": "off",
 		"@typescript-eslint/no-empty-interface": "off",
 		"@typescript-eslint/explicit-member-accessibility": "error",
+		"@typescript-eslint/no-unused-vars": [
+			"error",
+			{
+				argsIgnorePattern: "^",
+				varsIgnorePattern: "^_",
+				caughtErrorsIgnorePattern: "^_",
+			},
+		],
 	},
+	overrides: [
+		{
+			files: ["src/test/**/*"],
+			rules: {
+				"@typescript-eslint/no-unused-vars": ["off"],
+			},
+		},
+	],
 };
```

Comment:
It would be helpful to duplicate your explanation of the linter changes from '.eslint.js' in this file.

Reply:
done.